src/basic/hostname-util.c

```diff
@@ -191,13 +191,21 @@ bool is_localhost(const char *hostname) {
                 endswith_no_case(hostname, ".localhost.localdomain.");
 }
 
+const char *etc_hostname(void) {
```

Comment:
Nowadays, we use `const char* func()` rather than `const char *func()` style.
Please move `*` here and below.